docs/hermes.md

```diff
@@ -26,13 +26,48 @@ pod 'libevent', :podspec => '../node_modules/react-native-macos/third-party-pods
 > Be sure to set your target's deployment target to at least 10.14 before running `pod install`
 
 ## Hermes on Windows:
-We are working on bringing Hermes support to React Native for Windows through the official [Microsoft fork of Hermes](https://github.com/microsoft/hermes-windows).
+Hermes is experimentally supported on Windows, generally providing better performance characteristics than the default Chakra engine.
 
-Hermes for React Native for Windows has been updated to support the latest version 0.4.3 and can be turned on with a build flag for experimental use while we complete the feature set and get it ready for shipping.
+### Enabling Hermes for new projects
+The easiest way to enable Hermes is to pass the `--useHermes` flag to `react-native-windows-init` when creating a new project. This will set up your project to use the Herems engine and to generate bundles as Hermes bytecode instead of JavaScript.

```

Comment:
The easiest way to enable Hermes is to pass the `--useHermes` flag to `react-native-windows-init` when creating a new project. This will set up your project to use the Hermes engine and to generate bundles as Hermes bytecode instead of JavaScript.